src/lib/plot.typ

```diff
@@ -225,20 +238,60 @@
     }
   }
 
-  // Prepare
```

Comment:
All changed because of indentation: I moved the plot including axes into an inner group to be able to query it's bounds.